LunaDll/LuaMain/LuaProxyFFI.cpp

```diff
@@ -739,6 +739,51 @@ typedef struct ExtendedBlockFields_\
     {
         return {gMouseHandler.GetX(), gMouseHandler.GetY()};
     }
+	FFI_EXPORT(void) LunaLuaSetFullscreen(bool enable)
+    {
+        // Toggling fullscreen without maximizing the window/double clicking the window.
+		if (gMainWindowHwnd != NULL)
+        {
+            WINDOWPLACEMENT wndpl;
+            wndpl.length = sizeof(WINDOWPLACEMENT);
+            if (GetWindowPlacement(gMainWindowHwnd, &wndpl))
+            {
+                if ((wndpl.showCmd == SW_MAXIMIZE) && (enable == false))
+                {
+                    ShowWindow(gMainWindowHwnd, SW_RESTORE);
+                } else {
+					if (enable == true) {
+						ShowWindow(gMainWindowHwnd, SW_MAXIMIZE);
+					}
+                }
+            }
+        }
+    }
+	FFI_EXPORT(void) LunaLuaToggleWindowFocus(bool enable)
+    {
+        // Setting the focus to the window
+		if (enable) {
+			gStartupSettings.runWhenUnfocused = true;
+        } else {
+			gStartupSettings.runWhenUnfocused = false;
+        }
+    }
+	FFI_EXPORT(bool) LunaLuaIsFullscreen()
```

Comment:
You still have a mix of tab indentation and space indentation in your code. It should be all spaces, not tabs.
Since you mentioned Notepad++ before, see also: [here](https://community.notepad-plus-plus.org/topic/16129/how-do-i-tell-notepad-to-use-spaces-to-auto-indent-instead-of-tabs) and [here](https://i0.wp.com/odederell3d.blog/wp-content/uploads/2018/11/untitled-12.jpg?resize=648%2C681&ssl=1)